docs/tutorials/fundamentals/viewer.md

```diff
@@ -421,26 +421,25 @@ viewer.camera.perspective = 45
 
 #### Roll dimensions
 
-The third button rolls the dimensions that are currently displayed in the viewer. 
-For example if you have a `ZYX` volume and are looking at the `YX` slice, this 
-will then show you the `ZY` slice. You can also right-click this button to pop-up 
-a widget that allows you to re-order the dimensions by drag-and-drop or lock a 
+The third button rolls the dimensions that are currently displayed in the viewer.
+For example if you have a `ZYX` volume and are looking at the `YX` slice, this
+will then show you the `ZY` slice. You can also right-click this button to pop-up
+a widget that allows you to re-order the dimensions by drag-and-drop or lock a
 dimension, by clicking on the padlock icon:
 
 ![image: roll dimensions widget with padlock icons](../assets/tutorials/dims_roll_lock_widget.png){ w=200px }
 
-Locking prevents a dimension from being rolled (reordered). This can be particularly 
-useful, for example, with a `3D+time` dataset where you may want to fix the time dimension, 
+Locking prevents a dimension from being rolled (reordered). This can be particularly
+useful, for example, with a `3D+time` dataset where you may want to fix the time dimension,
 while being able to roll through the spatial dimensions.
 
-
-The dimension order can also be checked programatically as follows:
+The dimension order can also be checked programmatically as follows:
 
 ```{code-cell} python
 # To get the current dimension order as tuple of int
 viewer.dims.order
 ```
-And then, changed programatically as follows:
+And then, changed programmatically as follows:
```

Comment:
All of the changes in this file are unrelated typo/spelling fixes. I can move them to a different PR.